docs/guides/routing.mdx

```diff
@@ -0,0 +1,47 @@
+---
+description: Learn how Clerk handles routing in your application.
```

Comment:
I believe we need a title here:
```suggestion
title: Routing in Clerk
description: Learn how Clerk handles routing in your application.
```

Reply:
we no longer need titles, they are set from the h1 https://github.com/clerk/clerk-marketing/commit/191a51d8f3dc4509ef50effdf8c19762e575f5bf